theia-rust-docker/install_node.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+# Install node and npm via nvm - https://github.com/creationix/nvm
```

Comment:
There is a little glitch when starting a Theia terminal that seems related to using `nvm`. The following printout is output each time:
`nvm is not compatible with the npm config "prefix" option: currently set to "/root/.nvm/versions/node/v10.16.0"
Run `npm config delete prefix` or `nvm use --delete-prefix v10.16.0 --silent` to unset it.`

Reply:
Will look into this

Reply:
Fixed